test_stack.py

```diff
@@ -0,0 +1,34 @@
+import io
+import sys
+
+import pytest
+
+from stack import Stack
+
+
+def test_push():
+    s = Stack([7, 3, "yes"])
+    s.push("please")
+
+    assert len(s) == 4
+
+
+def test_pop():
+    s = Stack([6, 2, 6, 1])
+
+    assert s.pop() == 1
+    assert len(s) == 3
```

Comment:
This is good, maybe do one more, to prove 2 in a row work.
```suggestion
    assert s.pop() == 1
    assert len(s) == 3
    assert s.pop() == 6
    assert len(s) == 2
```